Antonio_Ramos_Documentation

```diff
@@ -369,3 +369,15 @@ Tab2 = Tab2.astype(convert_dict)
 - changed pathway reference in Local_3Stage_kfunction.py to import new shapefile CA_ZCTA_Man2.0.shp
 
 - ran program Local_3Stage_kfunction.py for 2 simulations and it successfully saved a shapefile "Manufa_Emp_C000_Points.shp". Infinite simulation loop error now solved. 
+
+7/28/2020
+- ran program successfully for 499 simulations. Shapefile exported. However, there is an issue with the GoodLabs_New.shp file where the labs are not projecting correctly in ArcGis. Problem corrected by using GoodLabs_Project.shp instead. 
```

Comment:
Can we get rid of GoodLabs_New.shp then?

Reply:
Yeah we can. It doesn't contain any original data, it is just the output of a python script will wrote.

Reply:
Okay, please make sure you do get rid of it and fix up the documentation at some point. We want to reduce the possibilities for confusion in the future, and getting rid of unnecessary files helps with that.